boards/arm/mimxrt1020_evk/mimxrt1020_evk.dts

```diff
@@ -58,9 +58,11 @@
 };
 
 &flexspi0 {
+	reg = <0x402a8000 0x4000>, <0x60000000 0x800000>;
 	qspi0: qspi@60000000 {
```

Comment:
as a proper spi child, should `qspi@60000000` use the chip select instead of the memory mapped address?